### ISSUE TITLE
[mediaqueries-5] prefers-noruby

### ISSUE THREAD

**Issue by NagayamaToshiaki** [w3c/csswg-drafts#7024]
The `<rt>` element is often rendered as small letters, so sometimes it's hard to see the text. Hence there might be someone who wants ruby text to display larger text next to the rubied sentence with parenthesis.

So let me declare the prefers-noruby media query. With this, the contents inside `<ruby>` element is rendered as if those elements are normal contents.

**Comment by fantasai** [comment]
@NagayamaToshiaki I think this could be easily done as a browser preference or extension, I don't think it requires a media query.

**Comment by tabatkins** [comment]
Yes, this is one of those things where the UA should be able to handle things generically, so there's no need to ask site authors to adapt their site manually.
